package.json

```diff
@@ -6,18 +6,20 @@
     "type": "git",
     "url": "https://github.com/MetaMask/create-release-branch.git"
   },
-  "bin": "dist/cli.js",
+  "bin": {
+    "create-release-branch": "dist/cli.js"
```

Comment:
Not sure this part is necessary - npm should automatically make it available without the organization prefix.

Reply:
Are we sure about that? The docs say:

> If you have a single executable, and its name should be the name of the package, then you can just supply it as a string.

But the name of the executable is not the name of the package in this case.

Reply:
I guess the docs don't account for package namespaces. In practice the namespace does not get included in the binary name. You can check for yourself in `node_modules/.bin`. `auto-changelog` works this way for example, the package being called `@metamask/auto-changelog`.

Reply:
I mean, I guess it couldn't hurt to explicitly name the binary :shrug: probably.

Reply:
Ah, right. I keep forgetting we already have a package that does this. Interesting, okay.

Reply:
Yeah... I guess the alternative is to update the NPM documentation, but absent of that, I can imagine myself getting confused about this again in the future.

Reply:
... is our linter objecting to this? :thinking:

Reply:
For some reason, `package.json` gets changed while running the action. I'm not totally sure why...

Reply:
Oh, `yarn install` seems to change it. Well then!

Reply:
This is now simplified again.